collector.go

```diff
@@ -181,6 +190,7 @@ func (c Collector) measure() error {
 		var measurementsRootDistance []float64
 		var measurementsPrecision []float64
 		var measurementsLeap []float64
+		var measurementClockSource []uint32
```

Comment:
Since we only end up using one value, we don't need to collect a slice. This can just be a `uint32`.